File: Java-Programming-Practise/Java-tutorial-in-hindi/Do_While_Loop.java
package com.java;

import java.util.Scanner;

public class Do_While_Loop {
    public static void main(String[] args) {
        int input, i=1;

        Scanner scanner = new Scanner(System.in);

        System.out.println("How many time you want the loop to run? ");

        input = scanner.nextInt();

        do {
            System.out.println(i);
            i++;
        }
        while (input >= i);
    }
}

File: C-Programming-Practise/Programming-Tutorial-For-Beginners-Learn-C/do_while_loop.c
#include <stdio.h>
int main()
{
    int num=1, input;
    scanf("%d", &input);
    printf("how many times you want the loop to run?\n");

    do
    {
        printf("%d\n", num);
        num++;
    } while (num <= input);
    
    
    return 0;
}